src/praisonai/tests/unit/bots/test_outbound_messenger.py

```
@@ -96,18 +96,64 @@ def test_send_unresolvable_target_fails_cleanly():
 
-def test_send_with_media_notes_unsupported_but_delivers_text():
+def test_send_with_missing_media_delivers_text_and_notes_skip():
     router, sent = _make_router()
     origin = SessionSource(platform="telegram", channel_id="123")
     messenger = BotOutboundMessenger(router, origin=origin)
 
+    # A non-existent path is rejected by the delivery-path guard; text still
+    # delivers and the skip is reported truthfully.
     result = asyncio.run(
-        messenger.send("origin", "Report", media=["/tmp/r.pdf"])
+        messenger.send("origin", "Report", media=["/tmp/does-not-exist-xyz.pdf"])
```

Ruff (0.15.18)

[error] Probable insecure usage of temporary file or directory: "/tmp/does-not-exist-xyz.pdf"

(S108)